CheatSheets/windows-cheatsheet.md

```diff
@@ -68,6 +69,14 @@ created: 2022-10-20
 |` Win + Tab`| Open task view|
 |` Win + M`| Minimize all windows |
 |` Win + R`| Open the Run dialog box |
+|` Win + A`| Open the Action Center |
+|` Win + C`| Launch Cortona in listening mode (used for voice commands) |
+|` Win + S`| Search/Launch Cortona (listening mode) |
+|` Win + I`| Open the settings app |
+|` Win + Ctrl + D`| Create New Virtual Desktop |
+|` Win + Ctrl + F4`| Close the Current Desktop |
+|` Win + Ctrl + Left Arrow Key / Right Arrow Key`| Switch between Virtual Desktops |
```

Comment:
```suggestion
| `Win + Ctrl + Left Arrow Key` <br /> `Win + Ctrl + Right Arrow Key` | Switch between Virtual Desktops |
```
| Command | Description |
| ----------- | --------------|
| `Win + Ctrl + Left Arrow Key` <br /> `Win + Ctrl + Right Arrow Key` | Switch between Virtual Desktops |

Please, modify it like this.